pkg/devices/holding_register.go

```diff
@@ -45,6 +47,19 @@ var HoldingRegisterHandler = sdk.DeviceHandler{
 	},
 }
 
+// ReadOnlyHoldingRegisterHandler is a device handler used to read modbus
+// holding registers. No write method is available.
+var ReadOnlyHoldingRegisterHandler = sdk.DeviceHandler{
+	Name: handlerReadOnlyHoldingRegister,
+	BulkRead: func(devices []*sdk.Device) (contexts []*sdk.ReadContext, e error) {
+		managers, found := DeviceManagers[handlerHoldingRegister]
```

Comment:
Pretty sure this part is okay ```DeviceManagers[handlerHoldingRegister]``` but worth a look.

Reply:
shouldn't this be `handlerReadOnlyHoldingRegister` here?